src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -5,12 +5,23 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace System.Net.Http.WinHttpHandlerTests
 {
+    public class testHandler:WinHttpHandler
```

Comment:
@josguil Why is this classname `testHandler:WinHttpHandler` at all?  I.e. why is there a colon character in the name?